README.md

```diff
@@ -17,41 +19,31 @@
 * Create as many as desired with easy polyzone creation
 * Vehicle sale amount gets deposited into the cardealer society fund for private dealer
 
-**Planned Updates**
-* qbx_phone support to make payments
-
-**Preview header when near a vehicle at the public dealership:**
-
-![image](https://user-images.githubusercontent.com/57848836/138773379-836be2a6-a800-47a4-8037-84d9052a964c.png)
-
-**After pressing the focus key and selecting the preview header (default: LEFT ALT)**
+**Vehicle Configuration System**
+* Model Check: The system first looks for the vehicle model in the models table. If a matching model is found, its will be displayed.
 
-![image](https://user-images.githubusercontent.com/57848836/138770886-15e056db-3e57-43ea-b855-3ef4fd107acf.png)
+* Category Check: If the vehicle model is not found, the system then checks the categories table to see if a category matches the vehicle. Its will be displayed in menu if a match is found.
 
-**Configurable test drive times that automatically return the player**
-![20211025160757_1](https://user-images.githubusercontent.com/57848836/138771162-00ee2607-0b56-418b-848c-5d8a009f4acd.jpg)
-
-**Vehicle purchasing**
-![20211025160853_1](https://user-images.githubusercontent.com/57848836/138772385-ce16c0e6-baea-4b54-8eff-dbf44c54f568.jpg)
+* Default Settings: If neither the model nor the category is found, the system will use the default shop to ensure the vehicle is still displayed.
 
-**Private job-based dealership menu (works off closest player)**
+**Planned Updates**
+* qbx_phone support to make payments
 
-![image](https://user-images.githubusercontent.com/57848836/138772120-9513fa09-a22f-4a5f-8afe-6dc7756999f4.png)
+**After pressing the focus key and selecting the preview header (default: E)**
 
-**Financing a vehicle with configurable max payment amount and minimum downpayment percentage**
-![image](https://user-images.githubusercontent.com/57848836/138771328-0b88078c-9f3d-4754-a4c7-bd5b68dd5129.png)
+![image](https://github.com/user-attachments/assets/1c0181dc-0a23-4b24-ac5a-cb36b2eb5bf3)
 
-**Financing preview header**
+**Configurable test drive times that automatically return the player**
```

Comment:
This should go into the test drive section?

Reply:
this should be in preview section